site/pages/docs/versions/v4.0/v4.0.35-en.hbs

```diff
@@ -0,0 +1,105 @@
+---
+{
+	"title": "v4.0.35 - release notes",
+	"language": "en",
+	"description": "v4.0.35 - release notes - Version history - Web Experience Toolkit (WET)",
+	"altLangPrefix": "v4.0.35",
+	"dateModified": "2020-04-16"
+}
+---
+<p>Version 4.0.35</p>
```

Comment:
Missing the STR tag to the version
```suggestion
<p>Version 4.0.35 (STR)</p>
```
Refer to v4.0.33 for STR tags.

Reply:
Fixed